schema_based_on_diagram.sql

```diff
@@ -0,0 +1,68 @@
+
+CREATE TABLE patients (
+    id serial PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+
+);
+
+-- invoice table
+
+CREATE TABLE invoices (
+    id serial PRIMARY KEY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP
+    visit_date DATE,
+    medical_history_id INT REFERENCES medical_histories(id)
```

Comment:
- [ Required ] As part of the project requirements I think it would be better if you create an `index` on all the `foreign keys` in your schema. An index makes searches in a table faster compared to when an index is not defined, to add an index on this foreign key we would the example below. Please add an index on all the `foreign keys` in your schema.

## Example of creating an index
```SQL
CREATE INDEX ON invoices (medical_history_id);
```